### PR TITLE
DEVELOPER-3345 Fixed mobile search bar

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by adelasofia** [comment]
retest this please

File: _layouts/base.html.slim [hunk 1]
```diff
@@ -115,17 +115,12 @@
...
-                    input.user-search(type="text" placeholder="Enter your search term" id="search_list_text" name="q" maxlength="128" required)
-                    button(type="submit" id="search-button")
-                      i.fa.fa-search
             - unless page.url.include?('search')
-              div.search-wrapper.mobile
+              .search-wrapper
                 form.search-bar(accept-charset="UTF-8" action="#{site.base_url}/search/" method="get" role="search" type="submit")
                   input.user-search(type="text" placeholder="Enter your search term" id="search_list_text" name="q" maxlength="128" required)
                   button(type="submit" id="search-button")
+                    i.fa.fa-search
```
Comment (LightGuard) [inline _layouts/base.html.slim]: All of these changes need to happen in html.html.twig

**Comment by LightGuard** [comment]
@KamiQuasi mind giving this a once over as well?

**Comment by robpblake** [comment]
retest this please

**Comment by KamiQuasi** [comment]
@LightGuard and @adelasofia other than making the changes in the twig file, the styles and changes look good to me. Definitely improves the look of the mobile search.

**Comment by adelasofia** [comment]
retest this please

File: _layouts/base.html.slim (hunk 1 above)
Comment (adelasofia) [inline _layouts/base.html.slim]: @LightGuard I just added this 👍

**Comment by adelasofia** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

**Comment by adelasofia** [comment]
Approved by James C

**Pull request merged**